### PR TITLE
global: initial alembic recipes

### PR DESCRIPTION
**Pull request by jirikuncar**
Depends on inveniosoftware/invenio-db#59

- [x] add foreign key to `transactions` table

### CODE REVIEW

**Comment by jirikuncar** [comment]
LGTM (cc @lnielsen)

File: invenio_accounts/alembic/843bc79c426f_create_accounts_branch.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+"""Create accounts branch."""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '843bc79c426f'
+down_revision = None
+branch_labels = (u'invenio_accounts', )
```
Comment (tiborsimko) [inline invenio_accounts/alembic/843bc79c426f_create_accounts_branch.py]: Thanks for changing the branch prefixes.  Looks good to me, CC @lnielsen.

File: setup.py [hunk 2]
```diff
@@ -42,26 +42,27 @@
...
+    'Flask-CeleryExt>=0.2.2',
     'Flask-Mail>=0.9.1',
 ]
 
 extras_require = {
     ':python_version=="2.7"': ['ipaddr>=2.1.11'],
     'celery': [
-        'celery>=3.1.0',
+        'celery>=3.1.0,<4.0',
+        'kombu>=3.0.37,<3.1',
```
Comment (lnielsen) [inline setup.py]: Ditto here - Is kombu dependency needed (https://github.com/celery/celery/blob/3.1/requirements/default.txt#L3)?

**Comment by lnielsen** [comment]
LGTM, just check the need for kombu

File: setup.py (hunk 2 above)
Comment (jirikuncar) [inline setup.py]: I had problems with it in https://travis-ci.org/inveniosoftware/invenio-accounts/jobs/173960774 I can run it again ...

**Comment by lnielsen** [comment]
LGTM

**Pull request merged**